### ISSUE TITLE
Wrong position of NUnit1001 diagnostic for TestCase with four parameters or more

### ISSUE THREAD

**Issue by cbersch** [nunit/nunit.analyzers#914]
For the following code, the diagnostic is at the wrong position:
```csharp
public sealed class MyTest
{
    [TestCase("string", 12, 27, "other")]
    public void Foo(string actual, Type expected, int value, string other)
    {
    }
}
```
**Actual result (with 4.10.0):**

<img width="662" height="178" alt="Image" src="https://github.com/user-attachments/assets/27aeba18-7c9d-4e03-89f7-c0861b6361ae" />

**Expected result**
The squiggles should mark the `12`.

This seems to happen if the number of parameters is four or more

**Comment by manfred-brands** [comment]
Fixed by #915
